tests/Spool/MemorySpoolTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of Raven.
+ *
+ * (c) Sentry Team
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Raven\Tests\Spool;
+
+use PHPUnit\Framework\TestCase;
+use Raven\Configuration;
+use Raven\Event;
+use Raven\Spool\MemorySpool;
+use Raven\Transport\TransportInterface;
+
+class MemorySpoolTest extends TestCase
+{
+    /**
+     * @var MemorySpool
+     */
+    protected $spool;
+
+    protected function setUp()
+    {
+        $this->spool = new MemorySpool();
+    }
+
+    public function testQueueEvent()
```

Comment:
Please add a test that checks that the spooling is done in the correct order (FIFO)

Reply:
Isn't it done below in the `testFlushQueue` method verifying it with `withConsecutive`?